### DISCUSSION TITLE
Bidirectional relationships

### DISCUSSION THREAD

**Discussion by arcaweb-ch** [kurkle/chartjs-chart-sankey#90] [Q&A] [answered]
Hi, loving this library thank you for your effort!

Here is my issue: I have the same elements in both From and To, every element shares data with each other, eg:

```json
[
{"From": "a", "To": "b", "Flow":5}
{"From": "a", "To": "c", "Flow":10},
{"From": "b", "To": "a", "Flow":15},
{"From": "b", "To": "c", "Flow":20},
{"From": "c", "To": "a", "Flow":35},
{"From": "c", "To": "b", "Flow":2},
]
```

I am trying to put all elements on left and right sides to show relations, like this (sort of):
![487118_sankey](https://user-images.githubusercontent.com/43749906/227342921-c9128fcf-1634-40c1-be87-0b825c5b22b5.jpg)

Any idea on how to accomplish this?

**Comment by kurkle** [comment, marked as answer]
You just need to use labels, colors etc to make the different sides appear to be same things, while actually being different nodes.
https://codepen.io/kurkle/pen/eYvNXqM

**Reply by arcaweb-ch** [reply]
Thanks a lot

**Reply by arcaweb-ch** [reply]
@kurkle any reason there is a white space between some labels? Also (sorry for duble question) is chart title supported by the library?
